docs/source/fetching/persisted-queries.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: Persisted Queries
+description: Secure your graph while minimizing request latency
+---
+
+Apollo supports two separate but related features called **persisted queries** and **automatic persisted queries** (APQs).
+With both features, clients can execute a GraphQL operation by sending an operation's ID instead of the entire operation string. An operation's ID is the SHA256 hash of the full string. Querying by ID can significantly reduce latency and bandwidth usage for very large operation strings.
+
+## Differences between persisted queries and APQ
+
+The persisted queries feature requires operations to be preregistered in a **persisted query list** (**PQL**). 
+This allows the PQL to act as a safelist of trusted operations made by your first-party apps. As such, persisted queries is a security feature as much as a performance one.
+
+With APQs, if the server can't find the operation ID the client provides, the server returns an error indicating that it needs the full operation string. If the Apollo iOS client receives this error, it automatically retries the operation with the full operation string.
+
+If you _only_ want to improve request latency and bandwidth usage, APQ addresses your use case. If you _also_ want to secure your supergraph with operation safelisting, you should preregister trusted operations in a PQL.
+
+For more details on differences between persisted queries and APQ, see the [GraphOS persisted queries documentation](/graphos/operations/persisted-queries#differences-from-automatic-persisted).
+
+### Requirements
+
+Persisted queries is currently in [preview](/resources/product-launch-stages#preview) and has the following requirements:
+- Apollo iOS (v1.3.1+)
+- [Apollo Router](/router) (v1.25.0+)
+- [GraphOS Enterprise plan](/graphos/enterprise/)
+
+You can use APQ with the following versions of Apollo iOS, Apollo Server, and Apollo Router:
+- Apollo iOS (v1.0.0+)
+- [Apollo Server](/apollo-server/) (v1.0.0+)
+- [Apollo Router](/router) (v0.1.0+)
+
+> **Note:** You can use _either_ Apollo Server _or_ Apollo Router for APQs. They don't need to be used together.
+
+## Implementation steps
+
+Both persisted queries and APQs require you to configure code generation and how your client makes requests. If you intend to use persisted queries for safelisting, you also need to generate an operations manifest.
+
+We recommend you follow this order while implementing: 
+
+1. **For persisted queries only**: Generate and publish operation manifests
```

Comment:
Nitpick: from a client point of view, there is a single manifest so I would use singular form here. Of course users with multiple schemas would have multiple manifest but I think it's an edge case and we should optimize for the majority.